### PR TITLE
Improve readme

### LINKED ISSUE #49: Broken link in readme
From https://github.com/sympy/sympy_benchmarks/blob/master/README.rst

http://spacetelescope.github.io/asv is not working
Maybe it should be changed to https://asv.readthedocs.io/en/stable/?

### PR DESCRIPTION
**Pull request by sylee957**
Fixes #49 
https://github.com/sympy/sympy/pull/16652#issuecomment-484289323

We don't need to make a full tutorial here, but I think we need some strong guide for the sympy main developers, about how to run the benchmarks locally to compare across forks or such.

I have added some basic tutorials

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -18,9 +18,65 @@ Results are currently hosted in an ad hoc manner, you may find some results `her
...
+``conda install -c conda-forge asv``
+
+And then you may also need to install ``virtualenv``. Because we have
+``"environment_type": "virtualenv"`` in ``asf.conf.json``.
+
+``pip install virtualenv``
+
+or in conda environment, use
+
+``conda install -c conda-forge virtualenv`` to install the dependency.
```
Comment (moorepants) [inline README.rst]: I can't imagine that this is a good idea.

i would be better to recommend using conda environments in asf.conf.json.

Comment (bjodah) [reply README.rst]: probably, I can't remember why we opted to keep `virtualenv` as the default.

Comment (moorepants) [reply README.rst]: I just swap out the conf file on my benchmark machine and use conda. Here is an issue: https://github.com/sympy/sympy_benchmarks/issues/4

Comment (moorepants) [reply README.rst]: And there was this attempt: https://github.com/sympy/sympy_benchmarks/pull/42

Comment (bjodah) [reply README.rst]: Ah, now I remember. @sylee957 opened https://github.com/sympy/sympy_benchmarks/pull/54, I added the auto-selection logic to `run_benchmarks.sh`

**Pull request merged**